grayskull/pypi/pypi.py

```diff
@@ -596,8 +623,9 @@ def _get_metadata(self) -> dict:
             )
         print_requirements(all_requirements)
 
-        test_entry_points = PyPi._get_test_entry_points(metadata.get("entry_points"))
-        test_imports = PyPi._get_test_imports(metadata, pypi_metadata["name"])
+        test_entry_points = PyPi._get_test_entry_points(metadata.get("entry_points", []))
+        test_imports = PyPi._get_test_imports(metadata, metadata["name"]) #pypi_metadata changed to metadata
+        metadata["version"] = "1.0.0" #hardcoded
```

Comment:
Add an `if` check here to do it only for github origins (http/https).

Reply:
But by this time the name of the package has already been extracted and the 'http' is lost.